### PR TITLE
fix: [PIE-3776]: Fix Formik Wrap issue - Terraform and DeployServiceSpecifications

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by ishantmehtaharness** [comment]
retrigger jest

**Comment by ishantmehtaharness** [comment]
retrigger coverage

**Review by kapilkumawat86 — DISMISSED** [review]

File: src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInputStep.tsx [hunk 1]
```diff
@@ -45,106 +44,102 @@ export default function TerraformInputStep<T extends TerraformData = TerraformDa
...
-                index={index}
-                {...props}
-                onUpdate={onUpdateRef}
-                onChange={onChangeRef}
-              />
-            )
-          }
-          return <></>
-        })}
+    <FormikForm>
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInputStep.tsx:47]: Why is `FormikForm` not removed?
Can you please cherry-pick the commit from develop

Comment (bhavya-sinha) [reply src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInputStep.tsx:47]: Why is FormikForm not removed?
Can you please cherry-pick the commit from develop

Comment (ishantmehtaharness) [reply src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInputStep.tsx:47]: There are fields in this form
also, it was already there. discussed this with @sahithikolichala

Comment (ishantmehtaharness) [reply src/modules/75-cd/components/PipelineSteps/Common/Terraform/TerraformInputStep.tsx:47]: this is cherry-pick only
we removed formikform for DeployServiceSpecifications/DeployServiceSpecifications
not from Terraform

**Review by bhavya-sinha — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**